mks/model/stuff.py

```diff
@@ -98,10 +98,10 @@ def _process_response(self, resp: ElementTree) -> None:
                 self._base_paths['partner'] +
                 self._base_paths['gerelateerde'])
 
-            # self.gerelateerde_ouder = objectify.ObjectPath(
-            #     self._base_paths['ouders'] +
-            #     self._base_paths['gerelateerde'])
-            #
+            self.gerelateerde_ouder = objectify.ObjectPath(
+                self._base_paths['ouders'] +
+                self._base_paths['gerelateerde'])
```

Comment:
What data does `gerelateerde` include ?

Reply:
`self._gerelateerde_ouder` is internal, it references the 'gerelateerde' xml stuff in the stuf data.